packages/react-headless/src/lib/realtime.ts

```diff
@@ -1,14 +1,13 @@
-import mitt from 'mitt';
-
-import clientSettings from '../stores/clientSettings';
-import NotificationRepository from '../stores/notifications/NotificationRepository';
+import { mitt } from '../polyfills/mitt-module.js';
+import clientSettings from '../stores/clientSettings.js';
+import NotificationRepository from '../stores/notifications/NotificationRepository.js';
 
 export function getAuthHeaders() {
   const { apiKey, userEmail, userExternalId, userKey } = clientSettings.getState();
 
   const headers = {
     'x-magicbell-api-key': apiKey,
-    'x-magicbell-client-user-agent': `${__PACKAGE_NAME__}/${__PACKAGE_VERSION__}`,
```

Comment:
What's your oppinion on importing `package.json` and using the values defined there for substituting `__PACKAGE_NAME__` and `__PACKAGE_VERSION__`?

Same in [packages/react-headless/src/stores/clientSettings.ts](https://github.com/magicbell/magicbell-js/pull/391/files#diff-a8fafc2fd01a9b830440d276821ca44947f35200a1094398391a3965f2d5924f)

Here's ho it looks in #390:
https://github.com/magicbell/magicbell-js/blob/c88bf5820b9eee0f256233e31ce33dc8d5d1e8d8/packages/react-headless/src/lib/realtime.ts#L9-L12

Reply:
Agreed. I didn't like the package.json imports, because not all bundlers might support importing from json files, but I came up with something better by using https://npmjs.com/replace-in-file right after build.